searx/engines/libretranslate.py

```diff
@@ -31,13 +31,19 @@ def request(_query, params):
 
     params['method'] = 'POST'
     params['headers'] = {'Content-Type': 'application/json'}
+    params["raise_for_httperror"] = False
```

Comment:
On some queries I get sometimes HTTP exception . here is the traceback of a 502

```
  File "searxng/local/py3/lib/python3.12/site-packages/httpx/_models.py", line 749, in raise_for_status
    raise HTTPStatusError(message, request=request, response=self)
httpx.HTTPStatusError: Server error '502 Proxy Error' for url 'https://trans.zillyhuhn.com/translate'
```

To suppress those issues we should handle HTTP issues in the response method ...

Reply:
That's probably been because of some instance related issues at zillyhuhn.com, I don't think that it generally throws such exceptions when it's hosted properly.

Reply:
Yes, we can drop this .. 

but if we know zillyhuhn.com  is not hosted properly, we may replace zillyhuhn.com  by a different [mirrow](https://translate.terraprint.co/) .. by [libretranslate.eownerdead.dedyn.io](https://libretranslate.eownerdead.dedyn.io/)?

Reply:
That mirror doesn't seem to load at all for me :thinking: 

There probably are more people self-hosting public instances, they're just not all at that list.

Reply:
> That mirror doesn't seem to load at all for me

Oops I haven't tested :-o / sorry.